.github/actions/upload-artifacts-to-s3/action.yml

```diff
@@ -0,0 +1,48 @@
+name: Upload Artifacts to S3
+description: |
+  Uploads module artifacts to S3 bucket.
+
+inputs:
+  aws-access-key-id:
+    description: 'AWS Access Key ID'
+    required: true
+  aws-secret-access-key:
+    description: 'AWS Secret Access Key'
+    required: true
+  github-ref:
+    description: 'GitHub ref'
+    required: true
+  osnick:
+    description: 'OS Nickname'
+    required: false
+    default: ''
+
+runs:
+  using: composite
+  steps:
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v3
+      with: # todo: use role instead of access key
+        aws-access-key-id: ${{ inputs.aws-access-key-id }}
+        aws-secret-access-key: ${{ inputs.aws-secret-access-key }}
+        aws-region: "us-east-1"
+    - name: Upload artifacts to S3 - staging
+      shell: bash
+      run: |
+        echo ::group::install aws cli
```

Comment:
Do we need these "echo" ?

Reply:
It collapses all logs under this name when we have a couple of actions in same step it is best practice to use it